packages/lib/src/interactions/hooks.ts

```diff
@@ -52,7 +52,11 @@ export function useZoomOnSelection() {
 
       // Update camera scale first (since `moveCameraTo` relies on camera scale)
       const { width: zoomWidth, height: zoomHeight } = zoomBox.size;
-      camera.scale.set(zoomWidth / width, zoomHeight / height, 1);
+      camera.scale.set(
+        Math.max(zoomWidth, 1) / width,
+        Math.max(zoomHeight, 1) / height,
```

Comment:
If the user draws a zoom box with a width or height of 0, we force a width or height of 1. This avoid zooming to infinity.